optional_requirements.txt

```diff
@@ -3,4 +3,6 @@ aiohttp==3.7.4.post0
 slackclient==2.7.2
 responses==0.10.14
 google-cloud-translate==2.0.1
-paramiko>=2.8.0
\ No newline at end of file
+paramiko>=2.8.0
+boto3>=1.21.21
```

Comment:
In addition to `boto3` we will need to add `boto3-stubs-lite[essential]>=1.21.21` as an optional dependency to support type-hinting.